fixbackend/notification/email/templates/account_degraded.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+
+<div class="f-fallback">
+    <h1 style="margin-top: 0; color: #333333; font-size: 22px; font-weight: bold; text-align: left;" align="left">
+        Account {{ message.cloud_account_id }} degraded!
+    </h1>
+
+    <p style="font-size: 16px; line-height: 1.625; color: #333; margin: .4em 0 1.1875em;">
+        One of your accounts can't be collected and is degraded now! Please check the account {{
+        message.cloud_account_id }} for issues.
+    </p>
```

Comment:
We were not able to collect latest resource information for Account xxxx. Please ensure the account exists and that the necessary permissions are granted for access.

Please visit https://app.global.fixcloud.io/workspace-settings/accounts#tenant-id for more details.